README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+-Responsive addresses accessibility issues. Media queries are generally used, but widths are not hard-coded unlike adaptive.
+
 2. Describe what it means to be mobile first vs desktop first.
 
+-Mobile first means you design the mobile version first and adjust it to be response to desktop, and vice-versa.
+
 3. What does `font-size: 62.5%` in the `html` tag do for us when using `rem` units?
 
+-The 62.5% is the conversion ratio between px and rem
+
 4. How would you describe preprocessing to someone new to CSS?
```

Comment:
elaborate a bit more on here such as how is it making it easier to use? mixins? variables? just cleaner code? nesting?